File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.java
class Solution {
    public int solution(int[] arr1, int[] arr2) {
        int answer = 0;
        int num1 = 0;
        int num2 = 0;
        if(arr1.length>arr2.length) answer = 1;
        if(arr2.length > arr1.length) answer = -1;
        if(arr2.length == arr1.length){
            for(int i=0; i<arr1.length; i++){
                num1+=arr1[i];
            }
            for(int j =0; j<arr2.length; j++){
                num2+=arr2[j];
            }
            
            if(num1 > num2){
                answer = 1;
            }
            if(num2>num1){
                answer = -1;
            }
            if(num1==num2){
                answer =0;
            }
            
        }
        return answer;
    }
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.kt
class Solution {
    fun solution(arr1: IntArray, arr2: IntArray): Int {
        var answer: Int = 0
        var num1 = 0
        var num2 =0
        if(arr1.size > arr2.size) answer = 1
        if(arr1.size < arr2.size) answer = -1
        if(arr1.size == arr2.size){
            for(i in 0..arr1.size -1){
                num1+=arr1[i]
            }
            
            for(j in 0..arr2.size -1){
                num2 += arr2[j]
            }
            if(num1 > num2) answer =1
            if(num2>num1) answer =-1
            if(num1 == num2) answer =0
            
        }
        
           
        return answer
    }
}